### PR TITLE
Added state to WizardWindow, updated DiscoveryGroupTestCase

### PR DESCRIPTION
**Pull request by honza-kasik**

### CODE REVIEW

File: common/src/main/java/org/jboss/hal/testsuite/fragment/shared/modal/WizardWindow.java [hunk 1]
```diff
@@ -159,4 +222,38 @@ public void waitUntilFinished() {
         Graphene.waitGui().withTimeout(1200, TimeUnit.MILLISECONDS);
         Graphene.waitModel().until().element(root, selector).is().not().visible();
     }
+
+    /**
+     * Providing information about state of wizard (open/closed)
+     */
+    public static final class State {
```
Comment (honza-kasik) [inline common/src/main/java/org/jboss/hal/testsuite/fragment/shared/modal/WizardWindow.java:224]: WDYT @zebrik?

File: common/src/main/java/org/jboss/hal/testsuite/fragment/shared/modal/WizardWindow.java [hunk 2]
```diff
@@ -159,4 +217,38 @@ public void waitUntilFinished() {
...
+            Assert.assertEquals("Wizard was supposed to fail, the window should be open.",
+                    InnerState.OPEN, state);
+        }
+
+    }
+
+    /**
+     * Obtains {@link State} of wizard
+     */
+    public State getWizardState() {
```
Comment (zebrik) [inline common/src/main/java/org/jboss/hal/testsuite/fragment/shared/modal/WizardWindow.java]: I would suggest private modifier for this method.

**Review by zebrik — APPROVED** [review]
Just one minor recommendation.

File: common/src/main/java/org/jboss/hal/testsuite/fragment/shared/modal/WizardWindow.java (hunk 2 above)
Comment (honza-kasik) [reply common/src/main/java/org/jboss/hal/testsuite/fragment/shared/modal/WizardWindow.java]: Fixed, thanks!

**Comment by zebrik** [comment]
LGTM. Thx.

**Pull request merged**